lib/v2/kloteneranzeiger/maintainer.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    '/:category?': ['JustYannicc'],
```

Comment:
https://github.com/DIYgod/RSSHub/blob/93d9bb53ab6ba75f036f7b563b0ec10bd45fdc08/lib/v2/kloteneranzeiger/router.js#L2
```suggestion
    '/news': ['JustYannicc'],
```